app/static/assets/css/stylesheet.css

```diff
@@ -186,6 +186,10 @@ p {
   margin: 10px 0 15px 0;
 }
 
+a {
+  color: #159588;
```

Comment:
I feel like this is nitpicking but should we use the existing CSS class named "oppia-foundation-green-text" instead of altering the global `<a>`'s color?
```CSS
.oppia-foundation-green-text {
  color: #009688;
}
```

Reply:
Also I feel like the color value #159588 does not quite match the #009688 used elsewhere. Is it just me?